client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -227,6 +228,8 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 			return <LoadingEllipsis />;
 		}
 
+		dispatch( recordTracksEvent( 'calypso_site_migration_ready_screen', migrationTrackingProps ) );
```

Comment:
Same reason as above, if we put it here it will trigger several times during the render. We might want to put it in the component.